package.json

```diff
@@ -36,7 +36,7 @@
     "lodash": "^4.17.15",
     "luxon": "^1.22.0",
     "no-scroll": "^2.1.1",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.1",
```

Comment:
For some reason, my local environment was hanging on `node-sass` when running `yarn install`.  This doesn't happen on IMPL or prod, so I think this might be unique to me.

After trying a few things, I ran `yarn install node-sass` to fix the issue so that's why there's a minor version update here. There are no breaking changes.